src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -248,27 +223,23 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
             }
 
             if (error) {
-                reject(error);
+                return reject(error);
```

Comment:
Is this return needed? at the end you have this code wrapped by the Promise, so if you reject a response, then the Promise understand that is a returned response

Reply:
It's a good practice. This is better explained here: https://stackoverflow.com/questions/32536049/do-i-need-to-return-after-early-resolve-reject/